src/wasm.js

```diff
@@ -1,8 +1,8 @@
-const loader = require("assemblyscript/lib/loader");
```

Comment:
it should be `@assemblyscript/loader`. And installed via npm as  deependency `npm i @assemblyscript/loader`.